docs/setup/security/ldap.md

```diff
@@ -10,10 +12,77 @@ identity:
         baseDn: dc=snowowl,dc=b2international,dc=com
         rootDn: cn=admin,dc=snowowl,dc=b2international,dc=com
         rootDnPassword: <adminpwd>
+        userObjectClass: inetOrgPerson
+        roleObjectClass: groupOfUniqueNames
         userIdProperty: uid
+        permissionProperty: description
+        memberProperty: uniqueMember
         usePool: false
 ```
 
-At a minimum, you must set the realm type to `ldap`, specify the `url` of the LDAP server and set the `rootDnPassword` in the `snowowl.yml` configuration file. 
-Your users should be available under the specified `baseDn` entry, and also there should be an `cn=admin` entry to allow access for Snow Owl to read user data.
-By default Snow Owl expects that the username of a user is present in the `uid` property. You can change this in the `userIdProperty` setting.
\ No newline at end of file
+## Configuration
+
+The following configuration settings are supported:
+
+| Configuration |      Description      |
+|---------------|-----------------------|
+| uri |  The LDAP URI that points to the LDAP/AD server to connect to |
+| baseDn | The base directory where all entries in the entire subtree will be considered as potential matches for all searches |
+| rootDn | The user's DN who has access to the entire `baseDn` and read content from it |
+| rootDnPassword | The password of the `rootDn` user |
+| userObjectClass | The user object's class to look for when searching for user entries. Defaults to `inetOrgPerson` class. |
+| roleObjectClass | The role object's class to look for when searching for role entries. Defaults to `groupOfUniqueNames` class. |
+| userIdProperty | The userId property to access and read for the user's unique identifier. Usually their username or email address. Defaults to `uid` property. |
+| permissionProperty | A multi-valued property that is used to store permission information on a role. Defaults to the `description` property.  |
+| memberProperty | A multi-valued property that is used to store and retrieve user `dn`s that belong to a given role. Defaults to the `uniqueMember` property. |
+
+The default configuration values are selected to support both OpenLDAP and Active Directory without needing to customize the default schema that comes with their default installation.
+
+## Configure Authentication
+
+When users send their username and password with their request in the Authorization header, the LDAP security realm uses the provided username and password to first search for the user in the configured LDAP instance to get the user's `DN` and then user the received `DN` and the provided password to authenticate with the LDAP instance. 
```

Comment:
```suggestion
When users send their username and password with their request in the Authorization header, the LDAP security realm uses the provided username and password to first search for the user in the configured LDAP instance to get the user's `DN` and then use the received `DN` and the provided password to authenticate with the LDAP instance. 
```

Reply:
That should be `use`, not `user`. This sentence is a bit too long, I'll rewrite it quickly.